[DOCS] GX Cloud Architecture Supporting Content (#11439)
great-expectations/great_expectations: docs/docusaurus/docs/reference/learn/gx_in_your_data_pipeline/delivery.md

---
sidebar_label: 'Delivery'
title: 'Delivery'

> description: Learn how GX can alert you to shifts in data metrics before it reaches BI tools.

Reviewer: There's some number disagreement in this sentence and I think it's unclear what the pronoun "it" refers to. 

```suggestion
description: Learn how GX can alert you to shifts in data metrics before trusted data reaches BI tools.
```

### How can GX Cloud help solve these problems?

GX Cloud can be implemented to surface valuable business insights automatically. By integrating GX after data transformation is complete, but before the validated data reaches BI tools, organizations can monitor for significant changes in business metrics and generate alerts accordingly. For example, if a given product suddenly sees much higher sales day over day, this insight should be automatically flagged and shared with the marketing and finance teams. This enables them to investigate the underlying cause, such as a successful campaign or seasonal trend, and make informed decisions based on timely data.

![An example showing that daily row counts are consistent until a spike on the last day, indicating a data anomaly.](/img/integration-delivery.png)

> Beyond anomalies, organizations can also use trends in the validation results to gain insights. For example, having confidence in accurate inventory and distribution data can help with supply chain management, reducing the risk of over- or understocking-goods, thereby decreasing operational costs over time. 

Reviewer: only the stranded prefix "over" needed a hyphen here so it would be read to mean "overstocking". There is no need for a hyphen between "understocking" and "goods" since those words are not acting as a phrasal noun here. 

```suggestion
Beyond anomalies, organizations can also use trends in the validation results to gain insights. For example, having confidence in accurate inventory and distribution data can help with supply chain management, reducing the risk of over- or understocking goods, thereby decreasing operational costs over time. 
```

---
sidebar_label: 'Delivery'
title: 'Delivery'

> ---

Reviewer: This page should have a `description:` in the frontmatter for SEO

> Data stakeholders are responsible for monitoring and interpreting business metrics. These individuals may span across different departments such as marketing, finance, operations, and may not all possess a technical background. One of the key challenges is detecting when shifts in data reflect real changes in business operations and outcomes. Often, these shifts go unnoticed until it's too late to respond effectively. Another issue is ensuring that the right non-technical users are alerted in a timely and relevant manner. Without targeted, contextual alerts, important insights may not reach the decision-makers who can act on them.

Reviewer: missing and

```suggestion
Data stakeholders are responsible for monitoring and interpreting business metrics. These individuals may span across different departments such as marketing, finance, and operations, and may not all possess a technical background. One of the key challenges is detecting when shifts in data reflect real changes in business operations and outcomes. Often, these shifts go unnoticed until it's too late to respond effectively. Another issue is ensuring that the right non-technical users are alerted in a timely and relevant manner. Without targeted, contextual alerts, important insights may not reach the decision-makers who can act on them.
```

> Data stakeholders are responsible for monitoring and interpreting business metrics. These individuals may span across different departments such as marketing, finance, operations, and may not all possess a technical background. One of the key challenges is detecting when shifts in data reflect real changes in business operations and outcomes. Often, these shifts go unnoticed until it's too late to respond effectively. Another issue is ensuring that the right non-technical users are alerted in a timely and relevant manner. Without targeted, contextual alerts, important insights may not reach the decision-makers who can act on them.

Reviewer: The phrasing "one of the key challenges" feels a bit off to me because the concept of challenges isn't brought up before this. 

here's a suggestion that I think flows more smoothly

```suggestion
Data stakeholders are responsible for monitoring and interpreting business metrics. These individuals may span across different departments such as marketing, finance, operations, and may not all possess a technical background. One of their key challenges is detecting when shifts in data reflect real changes in business operations and outcomes. Often, these shifts go unnoticed until it's too late to respond effectively. Another issue is ensuring that the right non-technical users are alerted in a timely and relevant manner. Without targeted, contextual alerts, important insights may not reach the decision-makers who can act on them.
```

> Data stakeholders are responsible for monitoring and interpreting business metrics. These individuals may span across different departments such as marketing, finance, operations, and may not all possess a technical background. One of the key challenges is detecting when shifts in data reflect real changes in business operations and outcomes. Often, these shifts go unnoticed until it's too late to respond effectively. Another issue is ensuring that the right non-technical users are alerted in a timely and relevant manner. Without targeted, contextual alerts, important insights may not reach the decision-makers who can act on them.

Reviewer: re "Another issue is ensuring that the right non-technical users are alerted in a timely and relevant manner. " earlier this paragraph says stakeholders " may not all possess a technical background" which implies that some stakeholders are technical users. I think the key characteristic of folks who need alerts is that they are stakeholders, not their level of technical expertise.

```suggestion
Data stakeholders are responsible for monitoring and interpreting business metrics. These individuals may span across different departments such as marketing, finance, operations, and may not all possess a technical background. One of the key challenges is detecting when shifts in data reflect real changes in business operations and outcomes. Often, these shifts go unnoticed until it's too late to respond effectively. Another issue is ensuring that the right stakeholders are alerted in a timely and relevant manner. Without targeted, contextual alerts, important insights may not reach the decision-makers who can act on them.
```

### How can GX Cloud help solve these problems?

> GX Cloud can be implemented to surface valuable business insights automatically. By integrating GX after data transformation is complete, but before the validated data reaches BI tools, organizations can monitor for significant changes in business metrics and generate alerts accordingly. For example, if a given product is selling 50% better in this current month than the previous month, this insight should be automatically flagged and shared with the marketing and finance teams. This enables them to investigate the underlying cause, such as a successful campaign or seasonal trend, and make informed decisions based on timely data.

Reviewer: the example ". For example, if a given product is selling 50% better in this current month than the previous month," feels a bit odd because it doesn't match what's shown in the following diagram. I suggest revising this to match the details of the narrative in the diagram.

![An example showing that daily row counts are consistent until a spike on the last day, indicating a data anomaly.](/img/integration-delivery.png)

> *GX Cloud can validate that there are no anomalies to the row count. In the example above, the row count on 1/4 shows a spike (i.e. an anomaly), which may indicate that a marketing campaign is succeeding.*

Reviewer: the preposition "to" feels off to me
missing comma

```suggestion
*GX Cloud can validate that there are no anomalies in the row count. In the example above, the row count on 1/4 shows a spike (i.e., an anomaly), which may indicate that a marketing campaign is succeeding.*
```

> *GX Cloud can validate that there are no anomalies to the row count. In the example above, the row count on 1/4 shows a spike (i.e. an anomaly), which may indicate that a marketing campaign is succeeding.*

Reviewer: This feels a bit repetitive of the preceding paragraph. I suggest removing this caption in favor of making the example in the preceding paragraph align with what's shown in the diagram.

> Beyond anomalies, organizations can also use trends in the validation results over time to gain insights. For example, having confidence in accurate sales data can help with forecasting inventory demands, reducing the risk of over or understocking goods, thereby decreasing operational costs over time. 

Reviewer: missing dash on stranded prefix

```suggestion
Beyond anomalies, organizations can also use trends in the validation results over time to gain insights. For example, having confidence in accurate sales data can help with forecasting inventory demands, reducing the risk of over- or understocking goods, thereby decreasing operational costs over time. 
```

> Beyond anomalies, organizations can also use trends in the validation results over time to gain insights. For example, having confidence in accurate sales data can help with forecasting inventory demands, reducing the risk of over or understocking goods, thereby decreasing operational costs over time. 

Reviewer: I find the use of "over time" a bit confusing here because the anomaly detection diagram above is literally showing a trend over time while the custom sql diagram below does not show a trend over time

I suggest removing "over time"

I also find the mention of "sales data" to be a bit disconnected from the illustrated "fulfillment stats" table. I suggest making the prose more aligned with the diagram

```suggestion
Beyond anomalies, organizations can also use trends in the validation results to gain insights. For example, having confidence in accurate inventory and distribution data can help with supply chain management, reducing the risk of over or understocking goods, thereby decreasing operational costs over time. 
```

> ![The statistics for the average processing time for fulfillment with a single row showing much higher than the others.](/img/integration-expectai.png)

Reviewer: This alt text is insufficiently descriptive, it doesn't mention ExpectAI which is shown in the diagram. Suggestion for more descriptive alt text

```suggestion
![ExpectAI performs deep analysis on a given Data Asset to set Expectations based on patterns in the data. For example, ExpectAI might suggest a Custom SQL test that average processing days for order fulfillment should be roughly proportional to the quantity of items shipped.  Outliers from the generally observed trend will be flagged for investigation.](/img/integration-expectai.png)
```

> *In the example above, GX Cloud can detect that the average processing day shows a large spike, which can trigger an investigation.*

Reviewer: This feels a bit repetitive of the alt text and preceding paragraph. I suggest removing it. 

```suggestion
```
